src/thunderbots/software/ai/hl/stp/evaluation/enemy_threat.cpp

```diff
@@ -0,0 +1,169 @@
+#include "ai/hl/stp/evaluation/enemy_threat.h"
+
+#include <deque>
+
+#include "ai/hl/stp/evaluation/intercept.h"
+#include "ai/hl/stp/evaluation/possession.h"
+#include "ai/hl/stp/evaluation/team.h"
+#include "ai/world/world.h"
+#include "geom/util.h"
+#include "shared/constants.h"
+
+std::optional<std::pair<int, std::optional<Robot>>> Evaluation::getNumPassesToRobot(
+    const Robot& initial_passer, const Robot& final_receiver, const Team& team,
+    const std::vector<Robot>& other_robots)
```

Comment:
Would `other_robots` ever be anything other then the enemy team?

Reply:
I don't think so. After some thought I'll change this to another `Team`.